### PR TITLE
Hw1 swagger and api

### PR DESCRIPTION
**Pull request by Yuliia9800**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,15 @@
...
+		"start:dev": "nodemon --watch src"
 	},
 	"author": "",
 	"license": "ISC",
-	"dependencies": {}
+	"dependencies": {
+		"express": "^4.18.2",
+		"nodemon": "^3.0.1",
+		"swagger-ui-express": "^5.0.0",
+		"yaml": "^2.3.2"
```
Comment (vasylkivskyy) [inline package.json]: Why do you need yaml package?

Comment (vasylkivskyy) [reply package.json]: nodemon package is intended for local development, so it should be in the DevDependencies section of the package.json.

File: src/swagger.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (vasylkivskyy) [inline src/swagger.json]: The instruction for the homework specifies using Swagger js docs for generating API (https://www.npmjs.com/package/swagger-jsdoc)

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+const app = express();
+const PORT = 3000;
+
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+
+app.get('/health-check', (req, res) => {
+	res.json({ status: 'Server is running' });
+});
+
+app.use('/not-found', (req, res) => {
```
Comment (vasylkivskyy) [inline src/index.js]: For a more general and standardized approach, it's preferable to use this option:
app.use((req, res, next) => {
  res.status(404).json({ error: 'Not Found' });
});
It's a cleaner and more maintainable solution when error handling is desired for undefined routes.

**Review by vasylkivskyy — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Yuliia9800) [reply package.json]: > Why do you need yaml package?

Forgot to delete, was trying to parse yaml file

**Review by vasylkivskyy — APPROVED** [review]

**Pull request merged**